q2_feature_table/tests/test_merge.py

```diff
@@ -286,15 +286,20 @@ def test_merge_seqs(self):
 class MergeFeatureTaxonomyTests(unittest.TestCase):
     # More extensive testing is performed in MergeFeatureDataTests, which
     # tests the shared private API.
+    # This tests a specifically FeatureData[Taxonomy]-like dataframe
+    # and ensures delivery in valid format (Taxon column first)
 
     def test_merge_taxa(self):
         # this test calls the public API directly
-        d1 = pd.Series(['a;b;c;d', 'a;b;c;e'], index=['f1', 'f2'])
-        d2 = pd.Series(['a;b;c;d', 'a;b;c;e'], index=['f1', 'f3'])
+        d1 = pd.DataFrame([('a;b;c;d', '1.0'), ('a;b;c;e', '1.0')],
+                          index=['f1', 'f2'], columns=['Taxon', 'Confidence'])
+        d2 = pd.DataFrame([('1.0', 'a;b;c;d'), ('1.0', 'a;b;c;e')],
```

Comment:
Can we make `f2` slightly different here, that way the `.combine_first` behavior is clear in the test?

Reply:
👍